keps/sig-node/4569-cgroup-v1-deprecation/README.md

```diff
@@ -0,0 +1,651 @@
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+This feature can be enabled or disabled using the `RemovedCgroupV1` feature gate. Disabling the feature gate will allow clusters to continue using cgroup v1.
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+-->
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: `RemovedCgroupV1`
+  - Components depending on the feature gate: Kubelet
+  - Will enabling / disabling the feature require downtime of the control
+    plane? Yes. Flag must be set on kubelet start. To disable, kubelet must be
+    restarted. Hence, there would be brief control component downtime on a
+    given node.
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node?
+    Yes. See above; disabling would require brief node downtime.
+
+###### Does enabling the feature change any default behavior?
+
+<!--
+Any change of default behavior may be surprising to users or break existing
+automations, so be extremely careful here.
+-->
+
+No
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+<!--
+Describe the consequences on existing workloads (e.g., if this is a runtime
+feature, can it break the existing applications?).
+
+NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
+-->
+
+Yes
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+N/A
+
+###### Are there any tests for feature enablement/disablement?
+
+<!--
+The e2e framework does not currently support enabling or disabling feature
+gates. However, unit tests in each component dealing with managing data, created
+with and without the feature, are necessary. At the very least, think about
+conversion tests if API types are being modified.
+-->
+
+N/A.
+
+### Rollout, Upgrade and Rollback Planning
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout fail? Can it impact already running workloads?
+
+<!--
+Try to be as paranoid as possible - e.g., what if some components will restart
+mid-rollout?
+-->
+
+Kubelet already supports cgroup v1 and cgroup v2. During `alpha` stage this feature is disabled by default.
+
+In the `beta` stage it is enabled by default. When this feature is enabled and the host is still utilizing cgroup v1, the kubelet will refuse to start. It is recommended that cluster administrators verify their cluster nodes' compatibitily by using the metric `kubelet_cgroup_version` before proceeding with the upgrade.
+
+###### What specific metrics should inform a rollback?
+
+<!--
+What signals should users be paying attention to when the feature is young
+that might indicate a serious problem?
+-->
+
+`kubelet_cgroup_version` metric should be used to make sure cluster nodes are using cgroup v2.
+
```

Comment:
Maybe NA actually.

This metric would not tell you if you should rollback this feature.